src/main.rs

```diff
@@ -1,18 +1,25 @@
 use std::sync::Arc;
 
 use clap::Parser;
+use nonempty::NonEmpty;
 use stratus::config::Config;
 use stratus::config::StorageConfig;
 use stratus::eth::evm::revm::Revm;
+use stratus::eth::evm::Evm;
 use stratus::eth::rpc::serve_rpc;
 use stratus::eth::storage::EthStorage;
 use stratus::eth::storage::InMemoryStorage;
 use stratus::eth::EthExecutor;
+use stratus::ext::new_tokio_runtime;
 use stratus::infra;
 use stratus::infra::postgres::Postgres;
 
-#[tokio::main]
-async fn main() -> anyhow::Result<()> {
+fn main() -> anyhow::Result<()> {
+    let runtime = new_tokio_runtime("tokio-main", 1, 1);
```

Comment:
The number of async and blocking threads should be configurable.

It is being set as 1 for now because it makes easier to detect deadlocks and other mistakes in async code.